woocommerce/woocommerce-gateway-stripe
# StripeLink: Add payment method option in admin

Fixes #2365 

## Changes proposed in this Pull Request:


In order to enable/disable StripeLink, a new option was added   in `Settings -> Payment Methods` section
The option is added under `Payments accepted on checkout`. 
The checkbox should be visible if UPE is enabled .
<img width="1178" alt="Screenshot 2022-06-29 at 09 40 52" src="https://user-images.githubusercontent.com/82826872/176369043-9ed3e1da-1b60-45b7-877f-4cc617aab322.png">





## Testing instructions
- run `wp option set _wc_stripe_feature_link 1` or `npm run wp option set _wc_stripe_feature_link 1 ` if running locally
- go to `Settings -> Payment Methods`
- Enable "the new checkout experience "
- Stripe Link payment method should be visible
- Enable/disable Stripe Link and refresh the page. The state should persist.




---


**Post merge**
